src/queueloader.cpp

```diff
@@ -69,6 +69,7 @@ nonstd::optional<QueueLoader::CategorizedDownloads> QueueLoader::categorize_down
 		case DlStatus::ALREADY_DOWNLOADED:
 		case DlStatus::READY:
 		case DlStatus::PLAYED:
+		case DlStatus::RENAME_FAILED:
```

Comment:
Please look through `TEST_CASE`s in test/queueloader.cpp and see if any of them need updating because of this change.

Reply:
Done